rotkehlchen/chain/ethereum/manager.py

```diff
@@ -306,26 +283,29 @@ def default_call_order(self, skip_etherscan: bool = False) -> List[NodeName]:
         """
         result = []
         if NodeName.OWN in self.web3_mapping:
-            result.append(NodeName.OWN)
+            # Assigning one is just a default since we always use it
```

Comment:
This code needs redesign. You have a result list, that is either empty or has own node. And then a selection list, which you later populate and randomize and then you return the sum of them. This makes no sense. The result should be the same but the `result` variable makes no sense anymore.